packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting-info/meetinginfov2.js

```diff
@@ -98,6 +126,15 @@ describe('plugin-meetings', () => {
         });
       });
 
+      it('create adoc meeting when conversationUrl passed with enableAdhocMeetings toggle', async () => {
+        sinon.stub(meetingInfo, 'createAdocSpaceMeeting').returns(Promise.resolve());
+
+        await meetingInfo.fetchMeetingInfo('conversationUrl', _CONVERSATION_URL_);
+
+        assert.calledWith(meetingInfo.createAdocSpaceMeeting, 'conversationUrl');
```

Comment:
probably worth also asserting that the normal meeting info api request is not sent